packages/teams-js/src/public/pages.ts

```diff
@@ -474,6 +523,9 @@ export namespace pages {
      * @param handler The handler to invoke when the user presses their Team client's back button.
      */
     export function registerBackButtonHandler(handler: () => boolean): void {
+      if (!isSupported()) {
```

Comment:
Interesting that this function doesn't have `ensureInitialized` at the beginning. I think that is an oversight and it should be added.